client/components/command-palette/index.tsx

```diff
@@ -151,6 +151,7 @@ const CommandPalette = () => {
 	const [ placeHolderOverride, setPlaceholderOverride ] = useState( '' );
 	const [ search, setSearch ] = useState( '' );
 	const [ selectedCommandName, setSelectedCommandName ] = useState( '' );
+	const [ lastEnteredSearch, setLastEnteredSearch ] = useState( '' );
```

Comment:
I would like that the variable name represent only when searching root commands.
Maybe `lastRootSearch` ? 🤔